cbo/management/commands/cbo_consistency_check.py

```
@@ -0,0 +1,27 @@
+import datetime
+
+from django.core.management.base import BaseCommand
+from myjobs.models import User
+from myjobs.models import Role
+from seo.models import Company
+from cbo.api import get_cbo_customer_id
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        users = User.objects.filter(roles__isnull=False).distinct()
+        cust_id = []
+        company_wo_custid = []
+        company_list = []
+        for user in users:
+            company_list = company_list + [r.company for r in user.roles.all()]
+        company_list = list(set(company_list))
+        for company in company_list:
+            try:
+                cust_id = get_cbo_customer_id(company)
```

Are you expecting exceptions here? A catch-all exception is usually a bad idea. If you put this in because you're seeing exceptions, we should look into that and handle it in the helper function.